docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -5,6 +5,93 @@ type = "docs"
+
+You can sign your plugin as a _public_ or a _private_ plugin. In both cases, you need to [create an account on Grafana.com](https://grafana.com/signup) and generate an API key with the `PluginPublisher` role. By creating an account, you can verify that you own the plugin that you want to sign.
+
+### Sign a public plugin
+
+Plugins signed under the community or commercial signature level are considered _public plugins_. Public plugins are published on [Grafana Plugin](https://grafana.com/plugins). For more information about installing public plugins, refer to [Install Grafana plugins]({{< relref "../../plugins/installation.md" >}}).
+
+1. Request a plugin signature level by sending an email to [XXX@grafana.com](mailto:XXX@grafana.com).
+
+1. Sign the plugin with the API key you just created. Grafana Toolkit creates a [MANIFEST.txt](#plugin-manifest) file in the `dist` directory of your plugin.
+
+   ```
+   export GRAFANA_API_KEY=<YOUR_API_KEY>
+   npx @grafana/toolkit plugin:sign
+   ```
+
+### Sign a private plugin
+
+If you're developing plugins for internal use only and don't want to make it public, you can sign it under a Private [signature level](#plugin-signature-levels).
+
+1. Sign the plugin with the API key you just created. Grafana Toolkit creates a [MANIFEST.txt](#plugin-manifest) file in the `dist` directory of your plugin.
+
+   The `rootUrls` flag accepts a comma-separated list of URLs for which the plugin can be used. The URLs need to match the [root_url]({{< relref "../../administration/configuration.md#root_url" >}}) setting.
+
+   ```
+   export GRAFANA_API_KEY=<YOUR_API_KEY>
+   npx @grafana/toolkit plugin:sign --rootUrls https://example.com/grafana
+   ```
+
+## Plugin signature levels
+
+To sign a plugin, you need to decide the _signature level_ you want to sign it under. The signature level of your plugin determines how you can distribute it.
+
+You can sign your plugin under three different _signature levels_.
+
+|**Plugin Level**|**Paid Subscription Required?**|**Description**|
+|---|---|---|
+|Private|No;<br>Free of charge|<p>You can create and sign a Private Plugin for any technology at no charge.</p><p>Private Plugins are for use on your own Grafana. They may not be distributed to the Grafana community, and are not published in the Grafana catalog.</p>|
+|Community|No;<br>Free of charge|<p>You can create, sign and distribute plugins at no charge, provided that all dependent technologies are open source and not for profit.</p><p>Community Plugins are published in the official Grafana catalog, and are available to the Grafana community.</p>|
+|Commercial|Yes;<br>Commercial Plugin Subscription required|<p>You can create, sign and distribute plugins with dependent technologies that are closed source or commercially backed, by entering into a Commercial Plugin Subscription with Grafana Labs.</p><p>Commercial Plugins are published on the official Grafana catalog, and are available to the Grafana community.</p>|
+
```

Comment:
It's probably worth reiterating here how to get set up to generate a community or commercial signature.